R/copy-model-from.R

```diff
@@ -222,17 +222,16 @@ copy_control_stream <- function(.parent_model_path, .new_model_path, .overwrite,
   if (.update_model_file) {
 
     # read parent control stream
-    mod_str <- .parent_model_path %>% read_file()
+    mod_str <- nmrec::read_ctl(.parent_model_path)
 
     # replace the $PROBLEM line(s)
```

Comment:
> Seems like it would be preferable to handle this in `bbi` [...]

What, in particular, do you have in mind?  

If you're saying that `bbi` should support multiple problems, that might be the ideal state, but I don't see that happening anytime soon.  And even if bbi/bbr did, nmrec would still need to be updated to handle multiple problems.

Or are you saying that bbi should give an informative error (for all relevant entry points: `run`, `summary`, ...)?  Even if that's the case, I don't think that's a guarantee that we don't hit into it here (it's a hand-edited file, and IIRC copying works fine with un-executed models).

So, I think the question is just, given that we think multiple problems aren't used in the wild with bbi/bbr, 1) are we okay with just letting an informative error bubble up from nmrec? or 2) do we want to catch it and warn that `.update_model_file = TRUE ` isn't in effect, and skip the "update model id" logic?

Reply:
> Or are you saying that bbi should give an informative error (for all relevant entry points: run, summary, ...)? Even if that's the case, I don't think that's a guarantee that we don't hit into it here (it's a hand-edited file, and IIRC copying works fine with un-executed models).

Yeah I was thinking more along these lines. But yeah fair point in that we could still run into the error here regardless of `bbi`'s handling. I did end up adding that catch in the latest commit though.